__tests__/services/users.service.spec.ts

```diff
@@ -5,46 +5,58 @@ import { UsersService } from '@services/users.service';
 import { factory } from 'typeorm-seeding';
 import { User } from '@entities/user.entity';
 import { getRepository, Repository } from 'typeorm';
-import { mockUpdateResult } from '../utils/mocks';
+import { mockCreateQueryBuilder, mockUpdateResult } from '../utils/mocks';
 import { HashInvalidError } from '@exception/users/hash-invalid.error';
 import { HashExpiredError } from '@exception/users/hash-expired.error';
 import * as faker from 'faker';
 import { UserNotFoundError } from '@exception/users/user-not-found.error';
 import { DatabaseError } from '@exception/database.error';
+import { JWTService } from '@services/jwt.service';
+import { ResetPassDTO } from '@dto/resetPassDTO';
+import { RecoverPassDTO } from '@dto/recoverPassDTO';
+import { EmailService } from '@services/email.service';
 
 let usersService: UsersService;
+let jwtService: JWTService;
 let user: User;
+let token: string;
 let userRepository: Repository<User>;
+let resetPassDTO: ResetPassDTO;
+let recoverPassDTO: RecoverPassDTO;
 
 describe('UsersService', () => {
   beforeAll( () => {
     usersService = Container.get(UsersService);
     userRepository = getRepository<User>(User);
+    jwtService = Container.get(JWTService);
   });
 
   it('all dependencies should be defined', () => {
     expect(usersService).toBeDefined();
     expect(userRepository).toBeDefined();
+    expect(jwtService).toBeDefined();
   });
 
   describe('comparePassword', () => {
     let userPassword: string;
+    let fakePassword: string;
 
     beforeEach(() => {
-      userPassword = hashSync('password', genSaltSync());
+      fakePassword = faker.lorem.word(6);
+      userPassword = hashSync(fakePassword, genSaltSync());
     });
 
-    it('checks that the password matches', () => {
-      const hashedPassword = 'password';
+    it('should return true when the passwords match', () => {
+      const hashedPassword = fakePassword;
       const result = usersService.comparePassword({
         password: hashedPassword,
         userPassword
       });
       expect(result).toBeTruthy();
     });
 
-    it("checks that the password don't match", () => {
-      const password = 'anotherpassword';
+    it("should return false when the passwords don't match", () => {
+      const password = faker.lorem.word(6);
       const result = usersService.comparePassword({ password, userPassword });
```

Comment:
I would rename password to `fakePassword` or something like that.

Reply:
You are right !